### PR TITLE
information-schema: add tidb_hot_regions_history docs and update tidb_hot_regions

### PR DESCRIPTION
**Pull request by IcePigZDB**
Signed-off-by: IcePigZDB <icepigzdb@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
* Add `information-schema/information-schema-tidb-hot-regions-history.md` to describes the usage of the new table`INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY`.
* Update the description of `INFORMATION_SCHEMA.TIDB_HOT_REGIONS` and add link to `INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- English version PR: pingcap/docs/pull/6624
- Other reference link(s):
    Relative docs:
    * Chinese vergion: pingcap/docs-cn/pull/7163
    * English vergion: pingcap/docs/pull/6627
    
    Feature tracking Issue: 
    * pingcap/tidb/pull/25281

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nolouch — APPROVED** [review]
lgtm. 
另外，这里也需要改一下？https://docs.pingcap.com/zh/tidb/stable/information-schema/#information-schema

**Comment by IcePigZDB** [comment]
> lgtm. 另外，这里也需要改一下？https://docs.pingcap.com/zh/tidb/stable/information-schema/#information-schema

Done, and CI should pass after the merge of  relative PD PRs.

**Comment by IcePigZDB** [comment]
In newest commit, I update the description of `INFORMATION_SCHEMA.TIDB_HOT_REGIONS` and add link to `INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY`.
* Relative discussion here https://github.com/pingcap/tidb/pull/30660

**Comment by nolouch** [comment]
PTAL @TomShawn

**Comment by TomShawn** [comment]
> PTAL @TomShawn

Tech review 是否完成了呢？如果完成了请 @en-jin19 看一下吧，谢谢~ 请注意新文档需要添加到 TOC 目录~

**Comment by IcePigZDB** [comment]
> Tech review 是否完成了呢？如果完成了请 @en-jin19 看一下吧，谢谢~ 请注意新文档需要添加到 TOC 目录~

TOC已经补充，Tech review 还差最后一个相关PR，合并后我@   en-jin19。

**Comment by IcePigZDB** [comment]
@en-jin19 技术PR已经全部合并进master了，PD和TiDB一共四个PR，thks~

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: 了解 information_schema 表 `TIDB_HOT_REGIONS_HISTORY`。
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+`TIDB_HOT_REGIONS_HISTORY` 表提供了关于历史热点 Region 的相关信息，这些信息由 PD 定期存储在本地，存储间隔为 [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval) 的值（默认值为 10m)，历史热点信息保持的期限为 [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-write-interval) 的值（默认值为 7days）。详情参见 [PD 配置文件描述](/pd-configuration-file.md#hot-regions-write-interval)。
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]: [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval) 这个是死链，官网上也搜不到。请确认下如何修改。@IcePigZDB

**Review by hfxsd — APPROVED** [review]
LGTM

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+* IS_LEADER：PEER 是否是 LEADER。
+* TYPE：热点 Region 的类型。
+* HOT_DEGREE：该热点 Region 的热度。
+* FLOW_BYTES：该 Region 内读写的字节数量。
+* KEY_RATE：该 Region 内读写的 Key 数量。
+* QUERY_RATE：该 Region 内读写的 Query 数量。
+
+> **注意：**
+>
+> + `TIDB_HOT_REGIONS_HISTORY` 表的 `UPDATE_TIME`、`REGION_ID`、`STORE_ID`、`PEER_ID`、`IS_LEARNER`, `IS_LEADER`、`TYPE` 字段会下推到 PD 服务器执行，所以为了降低使用该表的开销，必须指定搜索时间范围，然后尽可能地指定更多的条件。例如 `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`。
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:62]:
```suggestion
> + `TIDB_HOT_REGIONS_HISTORY` 表的 `UPDATE_TIME`、`REGION_ID`、`STORE_ID`、`PEER_ID`、`IS_LEARNER`、`IS_LEADER`、`TYPE` 字段会下推到 PD 服务器执行。所以为了降低使用该表的开销，必须指定搜索时间范围，然后尽可能地指定更多的条件。例如 `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`。
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+* 查询一段时间内的所有热点 Regions，替换 update_time 即可。
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT * FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00';
+  ```
+
+  > **Note:**
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:74]:
```suggestion
  > **说明：**
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+* HOT_DEGREE：该热点 Region 的热度。
+* FLOW_BYTES：该 Region 内读写的字节数量。
+* KEY_RATE：该 Region 内读写的 Key 数量。
+* QUERY_RATE：该 Region 内读写的 Query 数量。
+
+> **注意：**
+>
+> + `TIDB_HOT_REGIONS_HISTORY` 表的 `UPDATE_TIME`、`REGION_ID`、`STORE_ID`、`PEER_ID`、`IS_LEARNER`、`IS_LEADER`、`TYPE` 字段会下推到 PD 服务器执行。所以为了降低使用该表的开销，必须指定搜索时间范围，然后尽可能地指定更多的条件。例如 `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`。
+
+下面是一些常见的应用场景：
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:64]:
```suggestion
## 常见使用场景
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+* 查询一段时间内的所有热点 Regions，替换 update_time 即可。
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:66]:
```suggestion
* 查询一段时间内的所有热点 Regions。将以下语句中的 `update_time` 替换为实际所需的时间即可。
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

File: information-schema/information-schema-tidb-hot-regions-history.md (hunk 1 above)
Comment (easonn7) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]: Is the hot-regions-reserved-days link wrong ? please confirm.

Comment (IcePigZDB) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
`TIDB_HOT_REGIONS_HISTORY` 表提供了关于历史热点 Region 的相关信息，这些信息由 PD 定期存储在本地，存储间隔为 [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval) 的值（默认值为 10m)，历史热点信息保持的期限为 [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days) 的值（默认值为 7days）。详情参见 [PD 配置文件描述](/pd-configuration-file.md#hot-regions-write-interval)。
```

Comment (IcePigZDB) [reply information-schema/information-schema-tidb-hot-regions-history.md]: it's done, thank you.

File: information-schema/information-schema-tidb-hot-regions.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ summary: 了解 information_schema 表 `TIDB_HOT_REGIONS`。
 
 # TIDB_HOT_REGIONS
 
-`TIDB_HOT_REGIONS` 表提供了关于热点 Region 的相关信息。
+`TIDB_HOT_REGIONS` 表提供了关于当前热点 Region 的相关信息。历史热点信息可以在 `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)` 表查看。
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions.md:8]:
```suggestion
`TIDB_HOT_REGIONS` 表提供了关于当前热点 Region 的相关信息。历史热点信息可以在 [`TIDB_HOT_REGIONS_HISTORY`](/information-schema/information-schema-tidb-hot-regions-history.md) 表查看。
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: 了解 information_schema 表 `TIDB_HOT_REGIONS_HISTORY`。
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+`TIDB_HOT_REGIONS_HISTORY` 表提供了关于历史热点 Region 的相关信息，这些信息由 PD 定期存储在本地，存储间隔为 [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval) 的值（默认值为 10m)，历史热点信息保持的期限为 [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days) 的值（默认值为 7days）。详情参见 [PD 配置文件描述](/pd-configuration-file.md#hot-regions-write-interval)。
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
`TIDB_HOT_REGIONS_HISTORY` 表提供了关于历史热点 Region 的相关信息，这些信息由 PD 定期存储在本地，存储间隔为 [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval-从-v540-版本开始引入) 的值（默认值为 10m)，历史热点信息保持的期限为 [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days-从-v540-版本开始引入) 的值（默认值为 `7`），详情参见 [PD 配置文件描述](/pd-configuration-file.md#hot-regions-write-interval)。
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+* 查询一段时间内的所有热点 Regions。将以下语句中的 `update_time` 替换为实际所需的时间即可。
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT * FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00';
+  ```
+
+  > **说明：**
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:74]:
```suggestion
  > **注意：**
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: 了解 information_schema 表 `TIDB_HOT_REGIONS_HISTORY`。
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+`TIDB_HOT_REGIONS_HISTORY` 表提供了关于历史热点 Region 的相关信息，这些信息由 PD 定期存储在本地，存储间隔为 [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval-从-v540-版本开始引入) 的值（默认值为 10m)，历史热点信息保持的期限为 [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days-从-v540-版本开始引入) 的值（默认值为 `7`），详情参见 [PD 配置文件描述](/pd-configuration-file.md#hot-regions-write-interval)。
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
`TIDB_HOT_REGIONS_HISTORY` 表提供了关于历史热点 Region 的相关信息，这些信息由 PD 定期存储在本地，存储间隔为 [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval-从-v540-版本开始引入) 的值（默认值为 10m)，历史热点信息保持的期限为 [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days-从-v540-版本开始引入) 的值（默认值为 `7`），详情参见 [PD 配置文件描述](/pd-configuration-file.md#hot-regions-write-interval-从-v540-版本开始引入)。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@easonn7 PTAL again, thanks!

**Comment by easonn7** [comment]
LGTM

**Pull request merged**